kube-up.sh

```diff
@@ -38,11 +39,17 @@ fi
 
 echo "Cleaning up last kubernetes run (may ask for sudo)"
 if command -v docker-machine >/dev/null 2>&1; then
-    docker-machine ssh $DOCKER_MACHINE_NAME "mount | grep -o 'on /var/lib/kubelet.* type' | cut -c 4- | rev | cut -c 6- | rev | xargs --no-run-if-empty sudo umount"
+    docker-machine ssh $DOCKER_MACHINE_NAME "mount | grep -o 'on /var/lib/kubelet.* type' | cut -c 4- | rev | cut -c 6- | rev | sort -r | xargs --no-run-if-empty sudo umount"
     docker-machine ssh $DOCKER_MACHINE_NAME "sudo rm -Rf /var/lib/kubelet"
+    docker-machine ssh $DOCKER_MACHINE_NAME "sudo mkdir -p /var/lib/kubelet"
+    docker-machine ssh $DOCKER_MACHINE_NAME "sudo mount --bind /var/lib/kubelet /var/lib/kubelet"
```

Comment:
It sets up a bind mount which I can then change to shared below.  Basically the overall context is that since kubelet itself is in a container and it's trying to share emptyDirs, secrets, configMaps with other containers in its /var/lib/kubelet dir (which is where those things go) then they need to be in a mount marked as shared so that it can be shared across 2 containers. (kubelet and the container trying to use those mounts).